thriftbp/client_middlewares.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/reddit/baseplate.go/transport"
 )
 
+// HostHeader is the Thrift equivalent of the HTTP1 Host header and HTTP2 :authority pseudo-header.
+// Exposed for tests.
+const HostHeader = "thrift-hostname"
```

Comment:
we already have this defined: https://pkg.go.dev/github.com/reddit/baseplate.go/thriftbp#ThriftHostnameHeader